Sources/BraintreeCore/BTAPIClient.swift

```diff
@@ -26,7 +26,8 @@ import Foundation
 
     var http: BTHTTP?
     var graphQLHTTP: BTGraphQLHTTP?
-
+    var paypalHTTP: BTHTTP?
```

Comment:
super nit, but I think at one point we decided to case PayPal as `payPal` across the SDK. Do we want to make that update across this PR?

Reply:
Good catch, fixed in d944ed6fb987650ac4984f24d0dcfd9ccb6a56a2